package-lock.json

```
@@ -26458,4 +26458,4 @@
       "dev": true
     }
   }
-}
+}
```

not sure where this change comes from, but I think its a mistake

```suggestion
}

```

---

Not sure if this is the reason but @JulissaDantes can you make sure you have installed [EditorConfig](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig)?

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/d59306bd06a241083841c2e4a39db08e1f3722cc/.editorconfig#L10